components/Chatbar/components/ChatbarSettings.tsx

```diff
@@ -64,13 +61,25 @@ export const ChatbarSettings = () => {
 
       {/* {!serverSidePluginKeysSet ? <PluginKeys /> : null} */}
 
-      <SettingDialog
-        open={isSettingDialogOpen}
-        onClose={() => {
-          setIsSettingDialog(false);
-        }}
-        user={user}
-      />
+      <Transition
```

Comment:
Really not immediate, but for a background thought, we should make sure the more advanced transitions and animations don't cause compatibility issues across browsers, OSes and devices, esp. if we eventually expect people in the field (older and weaker devices) to use it.